lib/datadog/core/configuration/components.rb

```diff
@@ -345,6 +347,23 @@ def should_enable_gc_profiling?(settings)
               false
             end
           end
+
+          def print_new_profiler_warnings
+            if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.6')
+              Datadog.logger.warn(
+                'New Ruby profiler has been force-enabled. This feature is in alpha state. Please report any issues ' \
+                'you run into!'
```

Comment:
Should the message include a conveninence link to `https://github.com/DataDog/dd-trace-rb/issues/new`?

Reply:
Fixed in 960c4fa128a6dbf67b076baaa5849c17a0e77395